packages/presentation/lib/utils/extensions/extention_double.dart

```diff
@@ -0,0 +1,5 @@
+extension DoubleExtension on double {
+  double getTimeDouble() {
+    return double.parse((this / 2).toStringAsFixed(1));
```

Comment:
do it in mapper, don't create a separate extension for it. extension should be smth more general, like

num toPercent() => this * 100;

toPercent method can be used all over the project. getTimeDouble seems to be smth more specific

Reply:
moved to mapper